server/src/main/java/org/elasticsearch/action/admin/cluster/stats/RemoteClusterStatsRequest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.stats;
+
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+/**
+ * A request to get cluster level stats from the remote cluster.
+ */
+public class RemoteClusterStatsRequest extends ActionRequest {
+    private final String[] nodesIds;
```

Comment:
Does it really make sense to include node IDs in the remote-cluster request? It barely makes sense to restrict cluster-level stats to specific nodes in the local cluster, let alone in a remote.

Reply:
Well that is what I was told should be done but we can double-check. I will raise this question on the next PM sync.

Reply:
FWIW the telemetry use-case never specifies node IDs here

Reply:
Discussed and resolved to ignore nodes on remote requests.